### PR TITLE
Documentation for Ubuntu Bionic Packages.

### PR DESCRIPTION
**Pull request by Mistreated**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -23,6 +23,16 @@ For the following Linux distributions and package managers 0.8 packages are avai
 For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/show/home:bekun:devel/elektra)
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 
+### Ubuntu-Bionic
+
+To use the Ubuntu-Bionic packages following steps need to be made:
+
+1. Run `sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D919CE8B27A64C16656FCA9FF1532673651F9C6C` to obtain the key.
+
+2. Add `deb [trusted=yes] https://ubuntu-bionic-repo.libelektra.org/ bionic main` into `/etc/apt/sources.list`
```
Comment (markus2330) [inline doc/INSTALL.md:32]: `[trusted=yes]` is only needed if the key was not imported.

Comment (Mistreated) [reply doc/INSTALL.md]: Thank you.

File: doc/INSTALL.md [hunk 2]
```diff
@@ -23,6 +23,16 @@ For the following Linux distributions and package managers 0.8 packages are avai
 For [OpenSUSE, CentOS, Fedora, RHEL and SLE](https://build.opensuse.org/package/show/home:bekun:devel/elektra)
 Kai-Uwe Behrmann kindly provides packages [for download](http://software.opensuse.org/download.html?project=home%3Abekun%3Adevel&package=libelektra4).
 
+### Ubuntu-Bionic
+
+To use the Ubuntu-Bionic packages following steps need to be made:
```
Comment (mpranj) [inline doc/INSTALL.md:28]:
```suggestion
To use the Ubuntu-Bionic packages, the following steps need to be made:
```

**Comment by mpranj** [comment]
Thank you!

**Pull request merged**